src/components/ActionButton/ToggleActionButton.ts

```diff
@@ -0,0 +1,182 @@
+import { ComponentOptions, IResultsComponentBindings, Component, Initialization } from 'coveo-search-ui';
+import { ActionButton } from './ActionButton';
+
+export interface IToggleActionButtonOptions {
+    activatedIcon: string;
+    activatedTooltip: string;
+    icon: string;
+    tooltip: string;
+    click?: () => void;
+    activate?: () => void;
+    deactivate?: () => void;
```

Comment:
I would name them `onClick`, `onActivate`, `onDeactivate`

Reply:
I get your point, but I think the names are more coherent with the other components this way.